internal/cli/opsmanager/logs/decrypt.go

```diff
@@ -79,7 +79,7 @@ func DecryptBuilder() *cobra.Command {
 		Use:   "decrypt",
 		Short: "Decrypts a log file with the provided local key file or KMIP files.",
 		Example: `
-  $ mongocli ops-manager decrypt --localKey filePath --file logPath --out resultPath`,
+  $ mongocli ops-manager logs decrypt --localKeyFile filePath --file logPath --out resultPath`,
```

Comment:
Might be better to make it obvious the options takes a path by actually giving a path in the example (but using the right extension, I just guessed here!):

`$ mongocli ops-manager logs decrypt --localKeyFile /path/to/keyFile.pem --file /path/to/logFile.log --out /path/to/output/logFile.log`

Minor and not blocking.